Gruntfile.js

```diff
@@ -27,6 +27,24 @@ module.exports = function(grunt){
 
     // Copies templates and assets from external modules and dirs
     sync: {
+			assets_js: {
```

Comment:
the `assets_js` and `assets_images` sync tasks can just be removed and reverted back to what was there before
https://github.com/alphagov/govuk_prototype_kit/blob/master/Gruntfile.js#L29-L38